src/osemgrep/cli_scan/Ls_subcommand.mli

```diff
@@ -0,0 +1,19 @@
+(*
+   List files that semgrep treats as targets before rule-specific
+   or language-specific filtering.
+
+   This is an internal/experimental option for troubleshooting
+   currently implemented as a '--ls' flag of 'semgrep scan' for simplicity
```

Comment:
--ls or -x-ls?

Reply:
`--x-ls` (I originally wanted to call it just `--ls`)